src/daemon/custom_image_host.h

```diff
@@ -52,9 +53,13 @@ class CustomVMImageHost final : public VMImageHost
     std::vector<std::string> supported_remotes() override;
 
 private:
+    void update_manifest();
     CustomManifest* manifest_from(const std::string& remote_name);
+    std::chrono::seconds manifest_time_to_live;
```

Comment:
@townsend2010 Just noticed this is being default constructed (TTL=0). This was just an oversight, correct?

Reply:
@ricab,

Yes, you are correct, that is wrong.  In the `UbuntuVMImageHost` ctor, that value is passed in and assigned and the same should be done for the `CustomVMImageHost` or whatever common class you are implementing.